.github/workflows/sync-valint-version.yaml

```diff
@@ -0,0 +1,44 @@
+name: Sync Valint Version
+
+on:
+  pull_request:
+    types:
+      - closed
+
+permissions:
+  contents: write
+
+jobs:
+  sync_version:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master')
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Read version from file
+        id: read_version
+        run: echo "version=$(cat .version)" >> $GITHUB_OUTPUT
```

Comment:
Whats this file?

Reply:
It's pushed by the valint pipeline. I'll add a dummy one.